Dockerfile

```diff
@@ -1,5 +1,19 @@
 FROM afni/afni_make_build:AFNI_24.1.19
 
+# Workaround for https://github.com/afni/afni/issues/646
+# Placing under /opt/afni/data and setting AFNI_ATLAS_PATH so it would be
+# a generic soluiton for both make and cmake builds
+RUN wget -nv -O /tmp/afni_atlases.tgz https://afni.nimh.nih.gov/pub/dist/atlases/afni_atlases_dist_2024_0503.tgz && \
```

Comment:
this will be fixed up in

- https://github.com/afni/afni_proc_simple_bids_app/pull/3